storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/BlindedBlockMigration.java

```diff
@@ -85,16 +92,16 @@ private void moveHotBlocksToBlindedStorage() {
                 dao.finalizedUpdaterBlinded();
             KvStoreCombinedDaoUnblinded.HotUpdaterUnblinded unblindedUpdater =
                 dao.hotUpdaterUnblinded()) {
-          for (int i = 0; i < BLOCK_BATCH_SIZE && it.hasNext(); i++) {
+          for (int i = 0; i < blockBatchSize && it.hasNext(); i++) {
             final SignedBeaconBlock block = it.next();
             blindedUpdater.addBlindedBlock(block, block.getRoot(), spec);
             unblindedUpdater.deleteUnblindedHotBlockOnly(block.getRoot());
             counter++;
-          }
-          if (counter % 128 == 0 || counter == countBlocks) {
-            double percentCompleted = counter * 100.0 / countBlocks;
-            LOG.info(
-                "{} hot blocks moved ({} %)", counter, String.format("%.2f", percentCompleted));
+            if (counter % 128 == 0 || counter == countBlocks) {
+              double percentCompleted = counter * 100.0 / countBlocks;
+              LOG.info(
+                  "{} hot blocks moved ({} %)", counter, String.format("%.2f", percentCompleted));
```

Comment:
the status counter reporting just moved here so that it always works, regardless of whether the inner loop terminates at a specific point, which makes the reporting period easier to maintain